hathor/transaction/vertex_parser.py

```diff
@@ -26,10 +26,13 @@
 
 
 class VertexParser:
-    __slots__ = ('_settings',)
+    __slots__ = ('_settings', '_parse_genesis')
 
-    def __init__(self, *, settings: HathorSettings) -> None:
+    def __init__(self, *, settings: HathorSettings, parse_genesis: bool = True) -> None:
```

Comment:
I don't like the proposed solution to the problem because it is modifying the parser which, supposedly, has nothing to do with the vertex processing policies. I guess we should either modify the sync-v2 agent or the vertex handler.